PHP/MySQL.md

```diff
@@ -57,8 +64,5 @@ Tuning performance of Innodb
 
 Temporary tables
 
-
-
-
-
-
+# Links
+[Locks and transactions in MYSQL(In russian)](https://zinvapel.github.io/prog/database/2018/10/04/mysqlc-transaction/)
```

Comment:
Since all material and all links are in English I would suggest to an English article here

Reply:
I'll look for some English articles to add, but this particular one was too good to avoid.